src/test/regress/sql/local_shard_execution.sql

```diff
@@ -902,7 +902,7 @@ CALL regular_procedure('no');
 CALL regular_procedure('no');
 CALL regular_procedure('no');
 
-PREPARE multi_shard_no_dist_key(invite_resp) AS select * from event_responses where response = $1::invite_resp LIMIT 1;
+PREPARE multi_shard_no_dist_key(invite_resp) AS select * from event_responses where response = $1::invite_resp ORDER BY 1 DESC, 2 DESC, 3 DESC LIMIT 1;
```

Comment:
Wouldn't using only ORDER BY give a deterministic order always? Of course there is no harm in specifying "DESC".

Reply:
Yes, it would, I just like to order `DESC` :)